### ISSUE TITLE
Should we migrate from Travis to CircleCI?

### ISSUE THREAD

**Issue by zane** [probcomp/metaprob#135]
After Travis CI was sold to Idera [reports emerged that Idera was "gutting" the company](https://www.reddit.com/r/devops/comments/at3oyq/it_looks_like_ibera_is_gutting_travis_ci_just_a/) by laying off large numbers of Travis CI employees. We should consider whether we want to migrate over to Circle CI.

**Comment by joshuathayer** [comment]
Makes sense to me. Seems like Travis had [an extended outage](https://devclass.com/2019/03/29/travis-ci-users-left-hanging-as-platform-lies-down/) recently which people naturally tied to these senior layoffs. 

That said, I'm not sure where this should stand in our prioritized list- are there other more pressing issues now? And do we have an idea of what kind of effort this migration entails?
